### PR TITLE
Integrate register w bulmaswatch

### PR DESCRIPTION
**Pull request by andrewdevelopz**
### Project Changed

* [ ] API
* [x] APP

**- What I Did**

- Integrated `Slate` Bulmaswatch
- Added register form
- Added testing for register form

**- How I Did It**

Bulmaswatch
- yarn add bulmaswatch
- in index.js added 'import 'bulmaswatch/slate/bulmaswatch.min.css';

Register Form
- Added 'Users/Register' inside of Components folder
- The code concept is similar to Header.jsx but with form logic
- Validation logic was not added

**- How to Test/Verify It**

- Once you have the app running you will see the register form on the homepage
- Open up your browsers console and fill out the form
- You will see the results of what you typed in the console
- The test simply checks there is only 1 `h1` element and to verify the submit form button works

**- An Image to Make Us Giggle (not mandatory but encouraged)**

adgduoo

### CODE REVIEW

**Comment by tabuckner** [comment]
image
Nice 💪

File: broskichat-app/src/Components/Partials/Header.jsx [hunk 1]
```diff
@@ -71,6 +48,28 @@ class Header extends React.Component {
         }
       });  
   }
+
+  // When hamburger is clicked function
+  onClickHamburger = () => {
```
Comment (tabuckner) [inline broskichat-app/src/Components/Partials/Header.jsx:53]: @akeon-lee Much better! 👍

File: broskichat-app/src/Components/Users/Register/Register.jsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+    // Bind `this` to each method below
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  };
+
+  // Handles how to determine the event target value
+  handleInputChange(event) {
+    const target = event.target;
+    const value = target.type === 'checkbox' ? target.checked : target.value;
```
Comment (tabuckner) [inline broskichat-app/src/Components/Users/Register/Register.jsx:24]: nice ternary usage man

File: broskichat-app/src/index.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import 'bulma/css/bulma.css' // CSS Framework
```
Comment (tabuckner) [inline broskichat-app/src/index.js:3]: nice, i'm glad to see you also removed unused imports

**Review by tabuckner — APPROVED** [review]
Hell yeah, looks great man. Merge that shit!

**Review by Renestl — APPROVED** [review]

**Pull request merged**